crawl-ref/source/species-type.h

```diff
@@ -55,6 +55,7 @@ enum species_type
     SP_GNOLL,
     SP_FAIRY,
     SP_DUSK_WALKER,
+	SP_MOUNTAIN_DWARF,
     NUM_SPECIES,
```

Comment:
Funky indentation going on here...

Reply:
yeah, that's what happens when you leave a tab in somewhere